apps/judicial-system/web/src/routes/Court/RestrictionCase/Ruling/Ruling.tsx

```diff
@@ -123,23 +119,16 @@ export function getConclusionAutofill(
         isExtended:
           workingCase.parentCase &&
           isAcceptingCaseDecision(workingCase.parentCase.decision) &&
-          decision !== CaseDecision.ACCEPTING_ALTERNATIVE_TRAVEL_BAN
-            ? 'yes'
-            : '',
+          decision !== CaseDecision.ACCEPTING_ALTERNATIVE_TRAVEL_BAN,
         caseType:
           decision === CaseDecision.ACCEPTING_ALTERNATIVE_TRAVEL_BAN
             ? CaseType.TRAVEL_BAN
             : workingCase.type,
         validToDate: `${formatDate(validToDate, 'PPPPp')
           ?.replace('dagur,', 'dagsins')
           ?.replace(' kl.', ', kl.')}`,
-        hasIsolation:
-          isAcceptingCaseDecision(decision) && isCustodyIsolation
-            ? 'yes'
-            : 'no',
-        isolationEndsBeforeValidToDate: isolationEndsBeforeValidToDate
-          ? 'yes'
-          : 'no',
+        hasIsolation: isAcceptingCaseDecision(decision) && isCustodyIsolation,
+        isolationEndsBeforeValidToDate: isolationEndsBeforeValidToDate,
```

Comment:
```suggestion
        isolationEndsBeforeValidToDate,
```

Reply:
[b9da764](https://github.com/island-is/island.is/pull/7894/commits/b9da76477094fe1622451f8238ea898e741399b7)